### PR TITLE
KGE wikimedia example

### PR DESCRIPTION
**Pull request by sublimotion**
*Issue #, if available:*

*Description of changes:*
Adding Wikimedia example for DGL-KE

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: notebook-examples/kge_wikimedia.ipynb [hunk 1]
```diff
@@ -0,0 +1,1396 @@
+{
```
Comment (classicsong) [inline notebook-examples/kge_wikimedia.ipynb:1]: <p>I think we should deal with this Error output. And use build from source example to make dglke_score work.</p><p> </p><p>Others looks good to me. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/dgl-ke/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (sublimotion) [reply notebook-examples/kge_wikimedia.ipynb:1]: <p>@classicsong</p><p> </p><p>I have updated the notebook per feedback.</p><p> </p><p>Please take a look.</p>

File: notebook-examples/kge_wikimedia.ipynb [hunk 2]
```diff
@@ -0,0 +1,1398 @@
...
```
Comment (classicsong) [inline notebook-examples/kge_wikimedia.ipynb:1]: <p>It should be </p><pre class="ql-syntax" spellcheck="false">pushd dgl-ke;cd python;sudo python3 setup.py install;
</pre><p> </p><p>The master branch now include the inference instructions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/dgl-ke/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (classicsong) [inline notebook-examples/kge_wikimedia.ipynb:1]: <p>Is the result topk 10?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/dgl-ke/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

**Comment by classicsong** [comment]
There are two comments.
Others LGTM.

File: notebook-examples/kge_wikimedia.ipynb (hunk 2 above)
Comment (sublimotion) [reply notebook-examples/kge_wikimedia.ipynb:1]: <p>ok thanks. I have updated the install instructions per your feedback.</p>

Comment (sublimotion) [reply notebook-examples/kge_wikimedia.ipynb:1]: <p>Yes the result is top k. I have modified the argument from 5 to 10.</p>

**Pull request merged**